lib/Core/Algorithm/src/UpdateFunctions.c

```diff
@@ -666,8 +666,17 @@ static void _GenerateObservationCovariance_AHRS_Yaw()
      * CHANGED TO SWITCH BETWEEN GPS AND MAG UPDATES
      */
     if (useRTKHeading) {
-        gKalmanFilter.R[STATE_YAW] = gEKFInput.rtkHeading.headingAccuracy *
-            gEKFInput.rtkHeading.headingAccuracy;
+        // Limit the covariance to 2 degrees
+        //
+        // This accounts to the complete lack of synchronization, which on a system
+        // like tupan means something that 2-3 degree of error.
```

Comment:
I am very curious how you calculated / guessed this value, because I would have no idea how to guess it